test/unit-test/test_plt.ml

```diff
@@ -0,0 +1,28 @@
+open Base
+open Fialyzer
+module Etf = Obeam.External_term_format
+module E = Etf_util
+
+let%expect_test "Plt.of_file" =
+  let print filename =
+    Plt.of_file filename
+    |> [%sexp_of: (Plt.t, exn) Result.t]
+    |> Expect_test_helpers_kernel.print_s
+  in
+
+  print "plt/test01.plt";
```

Comment:
Could you write how to generate `test01.plt` as a comment?

Reply:
Wrote

Reply:
Thank you!